synapse/tools/genpkg.py

```diff
@@ -45,12 +53,28 @@ def loadPkgProto(path, opticdir=None):
         pkgdef['version'] = chopSemVer(pkgdef['version'])
 
     protodir = os.path.dirname(full)
+    pkgname = pkgdef.get('name')
 
     for mod in pkgdef.get('modules', ()):
         name = mod.get('name')
         with s_common.genfile(protodir, 'storm', 'modules', name) as fd:
             mod['storm'] = fd.read().decode()
 
+    for extmod in pkgdef.get('external_modules', ()):
```

Comment:
Do we need to keep the ``external_modules`` key in the finished ``pkgdef``?

Reply:
Nope, I'll remove it.